packages/core/chrome/core-chrome-browser-internal/src/ui/project/header.tsx

```diff
@@ -105,12 +180,31 @@ export const ProjectHeader = ({
               </ProjectNavigation>
             </Router>
           </EuiHeaderSectionItem>
-          <EuiHeaderSectionItem>{renderLogo()}</EuiHeaderSectionItem>
+
+          <EuiHeaderSectionItem>
+            <Logo
+              prependBasePath={prependBasePath}
+              application={application}
+              homeHref$={observables.homeHref$}
+              loadingCount$={observables.loadingCount$}
+            />
+          </EuiHeaderSectionItem>
+
+          <EuiHeaderSectionItem>
+            <HeaderNavControls navControls$={observables.navControlsLeft$} />
+          </EuiHeaderSectionItem>
+
           <EuiHeaderSectionItem>
             <HeaderBreadcrumbs breadcrumbs$={observables.breadcrumbs$} />
           </EuiHeaderSectionItem>
         </EuiHeaderSection>
+
         <EuiHeaderSection side="right">
+          <EuiHeaderSectionItem>
+            <HeaderNavControls navControls$={observables.navControlsCenter$} />
+            <HeaderNavControls navControls$={observables.navControlsRight$} />
```

Comment:
The global search bar is in the center slow, and the help menu is in the right.
<img width="1640" alt="image" src="https://github.com/elastic/kibana/assets/908371/fe2d35c5-567c-48bd-b5d0-6b42f50158f6">